Doc/library/ipaddress.rst

```diff
@@ -117,6 +116,16 @@ write code that handles both IP versions correctly.  Address objects are
    >>> ipaddress.IPv4Address(b'\xC0\xA8\x00\x01')
    IPv4Address('192.168.0.1')
 
+   .. versionchanged:: 3.10
+
+      Leading zeros are no longer tolerated and are treated as an error.
+      IPv4 address strings are now parsed as strict as glibc
+      :func:`~socket.inet_pton`.
+
+   .. versionchanged:: 3.8
```

Comment:
I understand that this change will be backported, since it's a little bit confusing to start documenting that 3.8 changed the behavior.